graphql/concrete_notification/fields.ts

```diff
@@ -39,26 +39,12 @@ export class ExtendedFieldsConcreteNotificationResolver {
         return await prisma.notification.findUnique({ where: { id: concreteNotification.notificationID } });
     }
 
-    @FieldResolver((returns) => String)
+    @FieldResolver((returns) => ConcreteNotification)
```

Comment:
changed to Object because MessageTemplate is a type and not a class and the linter complains in this case. tell me if there is a better way

Reply:
You can create a class for the type so that GraphQL has proper reflection and can generate the type definition from it. See https://github.com/corona-school/backend/blob/91031aa8762ca38eff877ecb0ca9bfa1c2436a2b/graphql/types/user.ts#L6 for an example

Reply:
I have replaced type with an abstract class for MessageTemplate